.github/workflows/test_unit.yml

```diff
@@ -19,13 +19,28 @@ jobs:
         uses: actions/checkout@v3
       - id: set-matrix
         run: ./scripts/resolve-modules.sh
-  test_unit:
-    name: Unit Tests
+  test:
+    name: Test
     needs: resolve-modules
     strategy:
       matrix: ${{ fromJson(needs.resolve-modules.outputs.matrix) }}
       fail-fast: false
     runs-on: ubuntu-latest
+    services:
```

Comment:
This was removed in https://github.com/cloudquery/cloudquery/pull/2074.

I suggest we keep it here at least until we ensure all CI passes without it